### ISSUE TITLE
HiSeq XTEN Quality encoding

### ISSUE THREAD

**Issue by AnnaMariaL** [magicDGS/ReadTools#24]
quality 42 ('K') occurs. 

https://www.biostars.org/p/179149/

Illumina info about their quality scores:
 http://www.illumina.com/content/dam/illumina-marketing/documents/products/technotes/technote_understanding_quality_scores.pdf

**Comment by magicDGS** [comment]
@AnnaMariaL, I added an option for allow high qualities in the developmental branch. Could you check if it is working before I realease the changes? Just do the following:

```
git clone https://github.com/magicDGS/ReadTools.git ReadToolsTest
mv ReadToolsTest
git checkout develop
mvn clean install
```

And them run the command that it was throwing the error before with and without the new `--allow-higher-qualities` option.

Thank you in advance.

**Comment by magicDGS** [comment]
Solved in commit efeab54b781c5f089242b9f5354bb30d900aa49d
